--- a/contrib/rm/Makefile.in
+++ b/contrib/rm/Makefile.in
@@ -84,13 +84,13 @@
 	test -e $(DESTDIR)$(libdir) || $(MKDIR_P) $(DESTDIR)$(libdir)
 	cp ${LIBNAME} $(DESTDIR)$(pkglibdir)
 	cp dmtcp_discover_rm $(DESTDIR)$(bindir)
-	cp dmtcp_rm_loclaunch.sh $(DESTDIR)$(bindir)
+	cp dmtcp_rm_loclaunch.sh $(DESTDIR)$(bindir)/dmtcp_rm_loclaunch
 
 uninstall:
 	echo " ( cd '$(DESTDIR)$(pkglibdir)' && rm -f $$files ) "; \
 	cd "$(DESTDIR)$(pkglibdir)" && rm -f $(LIBNAME); \
 	rm -f $(DESTDIR)$(bindir)/dmtcp_discover_rm 
-	rm -f $(DESTDIR)$(bindir)/dmtcp_rm_loclaunch.sh
+	rm -f $(DESTDIR)$(bindir)/dmtcp_rm_loclaunch
 
 
 ${LIBNAME}: $(OBJS) ${HEADERS}
